app/views/antminer.py

```diff
@@ -89,10 +104,11 @@ def miners():
                                 })
             temperatures.update({miner.ip: temps})
             fans.update({miner.ip: {"speeds": fan_speeds}})
-            hash_rates.update({miner.ip: ghs5s})
+            value, unit = update_unit_and_value(ghs5s, mapping[miner.model.model])
```

Comment:
If we get rid of the `mapping` dict we must update this line

Reply:
Mind elaborating how you thought on getting rid of the mapping? didn't follow

Reply:
And this could be like:

```
value, unit = update_unit_and_value(ghs5s, total_hash_rate_per_model['miner.model.model']['unit']  )
```

Reply:
done